template/docs/ods-stateless-components.md

```diff
@@ -0,0 +1,82 @@
+<img src="https://resource.alaskaair.net/-/media/2C1969F8FB244C919205CD48429C13AC" alt="Orion Design System Logo" title="Be the change you want to see" width="125" align="right" style="padding-left: 10px" />
+
+# ODS Stateless Components
+
+Orion Design System Stateless Components are framework agnostic reusable UI components intended for use across all internal and external facing applications across all of IT and e-commerce at Alaska Airlines.
+
+Contained within this repository will be the resources needed to build and deploy individual components and elements as they pertain to the Orion Design System (ODS)
+
+Please see all other documentation in regards to `CONTRIBUTING`, `CODE_OF_CONDUCT` and software `LICENSE` agreements of use.
+
```

Comment:
Add section ...

## What docs are there?

1. Babel support 
1. Browser support 
1. WCs and callbacks 
1. Custom properties 
1. WC Polyfill
1. Project setup
1. Tech details 
1. Tests